docs/src/observation_recipe.md

```diff
@@ -0,0 +1,100 @@
+# ObservationRecipe
+
+!!! warning
+    To enable this extension, use `using ClimaAnalysis` or `export
+    ClimaAnalysis`.
+
+When handling weather and climate data, it can be tedious and error-prone when
+setting up the observation for calibration with `EnsembleKalmanProcesses` (or
+`EKP` for short). As such, ClimaCalibrate provides recipes for setting up
+observations consisting of samples, noise covariances, names, and metadata.
+
+## How do I use this to set up observation for calibration with EKP?
+
+All functions assumes that any data preprocessing is done with `ClimaAnalysis`.
+
+### Covariance Estimators
+
+There are currently two covariance estimators which are
+`SeasonalDiagonalCovariance` and `SVDplusDCovariance` which are subtypes of
+`AbstractCovarianceEstimator`.
+
+For `SeasonalDiagonalCovariance`, the time series data of seasonal quantities
+should be used, while for `SVDplusDCovariance`, any time series data of any
+summary statistics can be used. Look at the documentation of these functions for
+more information.
+
+### Recommendations for data preprocessing
+
+The `OutputVar`s should represent time series data of summary statistics. For
+example, to compute seasonal averages of a `OutputVar`, one can use
+`ClimaAnalysis.average_season_across_time`, which will produce a `OutputVar`
+that can be used with either `SeasonalDiagonalCovariance` or
+`SVDplusDCovariance`.
+
+```julia
+import ClimaAnalysis
+ClimaAnalysis.average_season_across_time(var)
+```
+
+### Observation
+
+After preprocessing the `OutputVar`s, so that they represent time series data of
+summary statistics, one can use set up an `EKP.observation` as shown below.
+
+```julia
+import ClimaAnalysis
+import EnsembleKalmanProcesses as EKP
+import ClimaCalibrate
+import ClimaCalibrate.ObservationRecipe
+
+# vars are OutputVars preprocessed with ensure consistency with units, times,
+# and dimensions as the diagonstics produced from the model.
+# In this example, we want to calibrate with seasonal averages, so we use
+# ClimaAnalysis.average_season_across_time
+vars = ClimaAnalysis.average_season_across_time.(vars)
+
+# We choose SVDplusDCovariance. We need to supply the start and end dates of
+# the samples with `sample_dates`. It is helpful to use
+# `ClimaAnalysis.dates(var)` to find the correct dates.
+sample_dates = [
+        (Dates.DateTime(i, 12, 1), Dates.DateTime(i + 1, 9, 1)) for
+        i in 2007:2009
+    ]
```

Comment:
Maybe this could be a helper function since I expect some form of this will be repeated in calibration pipelines.